src/main/java/frc/robot/commands/climber/Climb.java

```diff
@@ -35,13 +36,17 @@ public Climb(DoubleSupplier speed, Climber climber, Intake intake) {
 
   @Override
   public void initialize() {
-    m_timer.start();
-    m_intake.setExtended(true);
+    if (m_intake.isExtended()) {
```

Comment:
`m_isIntakeExtended = m_intake.isExtended()`

Reply:
Resolved in commit 408960c

Reply:
**